src/arrayMethodJoin.js

```diff
@@ -3,9 +3,26 @@
 /**
  * Implement method join
  */
+
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```

We can add a default value to separator